Detectors/TPC/reconstruction/macro/findKrBoxCluster.C

```diff
@@ -0,0 +1,90 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file findKrBoxCluster.C
+/// \brief This macro retrieves clusters from Krypton and X-Ray runs, input tpcdigits.root
+/// \author Philip Hauer <philip.hauer@cern.ch>
+
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+#include "TCanvas.h"
+#include "TFile.h"
+#include "TTree.h"
+
+#include "TPCReconstruction/KrCluster.h"
+#include "TPCReconstruction/KrBoxClusterFinder.h"
+#include "TPCBase/Digit.h"
+
+#include <array>
+#include <iostream>
+#include <tuple>
+#include <vector>
+#endif
+
+void findKrBoxCluster_dev()

```

Comment:
The function should be called as the script. Please remove `_dev`.

Reply:
Oh no, sorry, you're right..fixed it!